content/handbook/product-development/how-we-work/product-development-flow/_index.md

```
@@ -0,0 +1,527 @@
+
+## 概要と哲学
+
+GitLab のプロダクトミッションは、ユーザーが愛し価値を感じてくれるプロダクトと体験を一貫して作ることです。このミッションを果たすには、アイデアからカスタマーバリューを提供するものへと変えるための、明確に定義され繰り返し可能なフローを持つことが重要です。なお、より広範な GitLab コミュニティから、プロセスのあらゆる時点でオープンソースのコントリビューションを受け付けることも重要です - それらはこのプロセスには必ずしも従いません。
+
+このページは、クロスファンクショナルな開発チームに私たちが期待する作業の進め方の進化的な説明であり、現在活用されているワークフローを反映しています。このページにおける必須のアクションやアウトカムはすべて、次のように示されます：
+
+> <i class="fab fa-gitlab fa-fw" style="color:rgb(252,109,38); font-size:1.25em" aria-hidden="true"></i> プロダクト開発フローにおける必須の側面を示します。
+
+機能開発は、指定されたアウトカムを達成するためにすべてのフェーズを通ることが期待されますが、残りのワークフローは、ベストプラクティス、ツール、推奨事項のセットと見なすべきです。私たちは、特定のプロダクト改善がすべてのフェーズを経る必要のないユニークなケースがあることを認識しています。デザインおよびエンジニアリングチームと整合しつつ、プロダクトマネージャーが最善の判断を使うことを信頼しています。このページの目標は、各フェーズで目標とすべき必須の **アウトカム**、およびこれらのアウトカムを達成するためにチームが採用できる戦略／戦術、**アクティビティ**を強調することで、ワークフローでチームをサポートすることです。さらに、このページは、追跡、検索、クロスファンクショナル協働の観点でプロダクトシステムを効率的に保つために、すべてのフェーズで必要な*最小*のアクション（ラベルなど）を明確にすることを目指しています。明確さを維持し混乱を避けるため、このページにはオプションのアクションを記載していませんが、ここに記載されていなくても、チームは計画用のラベルなどの追加のアクションを採用することを選択できます。
+
+チームがプロダクト開発フローを活用するなかで、特定の戦略／戦術がチームの成功に役立っていることに気づくかもしれません。そのため、カスタマー向けの価値ある機能を構築するための選択肢の堅牢なプレイブックを作成できるよう、このページへの MR を歓迎します。すべてのチームメンバーは、ベストプラクティスを共有するために、このページの[変更プロセス](#contributing-to-this-page) に従うことを推奨されます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)